### PR TITLE
Adding the `pymp-pypi` package to conda-forge.

### PR DESCRIPTION
**Pull request by classner**
Hi!

This is a PR to add the PyMP package to conda-forge (as requested in https://github.com/classner/pymp/issues/24 ). Tests run on my machine (they will not run on Windows as this package requires OS level fork support). As listed in the PR template, 
@conda-forge-admin, please ping team @conda-forge/help-python .

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by dopplershift — APPROVED** [review]

**Comment by classner** [comment]
Hi @dopplershift,
I see tests on Linux and MacOS failing because of a missing license file. I don't know why though, because the file definitely exists. Any tips?

Thanks,
Christoph

**Comment by xhochy** [comment]
The license file is not part of the `sdist`. You need to create a `MANIFEST.in` file which has an `include LICENSE` in it to ensure that the license file is part of the `sdist` tarball.

**Comment by classner** [comment]
All issues have been fixed on my end as far as I could, however some of the tests are still failing due to multiple imports of the package, for example here: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=363321&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1190 . Due to the way of how the package works with forking, this is not possible without errors.

File: recipes/pymp-pypi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - pip
+    - python >=2.6
+  run:
+    - python >=2.6
+
+test:
+  imports:
+    - pymp
+    - tests
```
Comment (xhochy) [inline recipes/pymp-pypi/meta.yaml:28]: The tests shouldn't be installed in global scope.

File: recipes/pymp-pypi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - pip
+    - python >=2.6
+  run:
+    - python >=2.6
+
+test:
+  imports:
+    - pymp
```
Comment (xhochy) [inline recipes/pymp-pypi/meta.yaml:27]: Would it work better if this would be called via `python -c "import pymp"`?

**Comment by classner** [comment]
@xhochy Great suggestions, works! Does this unblock the merge?

**Pull request merged**